lib/EnsEMBL/REST/Model/LDFeatureContainer.pm

```diff
@@ -116,11 +116,67 @@ sub fetch_LDFeatureContainer_slice {
   return $self->to_array($ldfc);
 }
 
+sub fetch_LDFeatureContainer_pairwise {
+  my ($self, $variation_name1, $variation_name2) = @_;
+  my $c = $self->context();
+  my $species = $c->stash->{species};
+
+  my $va = $c->model('Registry')->get_adaptor($species, 'Variation', 'Variation');
+  my $ldfca = $c->model('Registry')->get_adaptor($species, 'Variation', 'LDFeatureContainer');
+  my $pa = $c->model('Registry')->get_adaptor($species, 'Variation', 'Population');
+
+  my $ld_config = $c->config->{'Model::LDFeatureContainer'};
+  if ($ld_config && $ld_config->{use_vcf}) {
+    $ldfca->db->use_vcf($ld_config->{use_vcf});
+    $ldfca->db->vcf_config_file($ld_config->{vcf_config});
+    $ldfca->db->vcf_root_dir($ld_config->{dir}) if (defined $ld_config->{dir});
+  }
+
+  my @vfs = ();
+  foreach my $variation_name ($variation_name1, $variation_name2) {
+    my $variation = $va->fetch_by_name($variation_name);
+    Catalyst::Exception->throw("Could not fetch variation object for id: $variation_name.") if ! $variation;
+    my $vfs = $variation->get_all_VariationFeatures();
+    Catalyst::Exception->throw("Variant maps more than once to the genome.") if (scalar @$vfs > 1);
+    Catalyst::Exception->throw("Could not retrieve a variation feature.") if (scalar @$vfs == 0);
+    push @vfs, $vfs->[0];
+  }
+
+  my $population_name = $c->request->param('population_name');
+  if ($population_name) {
+    my $pa = $c->model('Registry')->get_adaptor($species, 'Variation', 'Population');     
+    my $population = $pa->fetch_by_name($population_name);
+    if (!$population) {
+      Catalyst::Exception->throw("Could not fetch population object for population name: $population_name");
+    }
+    my $ldfc;
+    try {
+      $ldfc = $ldfca->fetch_by_VariationFeatures(\@vfs, $population);
+    } catch {
+      Catalyst::Exception->throw("Something went wrong while fetching from LDFeatureContainerAdaptor");
+    };
+    return $self->to_array($ldfc)
+  }
+  # compute LD for all LD populations
+  my $ld_populations = $pa->fetch_all_LD_Populations;
```

Comment:
I think here it would raise a warning the population adaptor is undefined in the default case. By default, population name is unset hence line 147 would not be executed.